templates/apiary-bucket-policy.json

```diff
@@ -18,19 +18,32 @@
                 "arn:aws:s3:::${bucket_name}"
             ]
         },
-%{endif}
-%{if customer_principal != ""}
+%{if customer_condition == ""}
```

Comment:
if `customer_condition` is empty, will `customer_principal` definitely be defined?

Reply:
no
I switched to use nested if , all these policies wont be create if customer_principal is empty 
https://github.com/ExpediaGroup/apiary-data-lake/blob/bc750d59fbd45e8a69ebbf765d2effe51beb2139/templates/apiary-bucket-policy.json#L5

Reply:
Oh sorry, PR didn't show me that line.  I should have checked.